packages/yara/yara.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "Anton Kochkov <anton.kochkov@gmail.com>"
+authors: "Anton Kochkov <anton.kochkov@gmail.com>"
+homepage: "https://github.com/XVilka/yara-ocaml"
+bug-reports: "https://github.com/XVilka/yara-ocaml/issues"
+license: "MIT"
+dev-repo: "https://github.com/XVilka/yara-ocaml.git"
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "jbuilder" {build}
+  "ppx_deriving" {build}
+  "core"
+  "ctypes"
```

Comment:
I think you need a dependency on `ctypes-foreign` here too.

Reply:
Fixed, thank you.